docs/tutorials/intro_tutorial.ipynb

```diff
@@ -60,7 +60,13 @@
     "\n",
     "Create and activate a [virtual environment](http://docs.python-guide.org/en/latest/dev/virtualenvs/). *Python version 3.10 or higher is required*.\n",
     "\n",
-    "Install Mesa:\n",
+    "Install Mesa with the recommended dependencies.\n",
+    "\n",
+    "```bash\n",
+    "pip install --upgrade mesa[rec]\n",
+    "```\n",
+    "\n",
+    "If you already have all the dependencies installed or are just updating, you can install Mesa without the recommended dependencies:\n",
```

Comment:
How does this deconflict from the stable version (e.g. 2.3.4) and the pre-release? 

As one has [stable in the url](https://mesa.readthedocs.io/en/stable/tutorials/intro_tutorial.html) and one has [latest in the url](https://mesa.readthedocs.io/en/latest/tutorials/intro_tutorial.html). The challenge at this point is there so many changes model implementation will be pretty different.  

I think it would be too painful to try and keep the stable and the latest tutorials, but maybe explicitly call out of the user to check which version they are in???

I am not sure the right answer this is an interesting problem.

Reply:
It’s quite elegant: the stable one is using the latest stable release, so now 2.3.4. We generally point users to that.

The latest release is built from `main`, which would be updated if this PR got merged.

Once we then release a 3.0.0 stable release, the stable docs get switched over also.

Reply:
I understand that, but with `a5` intro tutorial switching to new `AgentSet` set up I think we need to explicitly call out there is a difference as that subtlety can be lost on users. For example, if I send out a link to latest to a new user but then they search Mesa later on they will likely get stable and the differences I think will only confuse them. 

Make sense?

Reply:
Yeah we could add notes to both that one is for Mesa 2.3.x and the other is for Mesa 3.0 pre-releases. We did that already with the Viz tutorial.